src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Shapes/Basic_Shapes.xaml.cs

```diff
@@ -333,6 +341,25 @@ private static byte[] RenderAsPng(FrameworkElement elt)
 
 			return stream.ToArray();
 		}
+#elif __SKIA__
+		private static byte[] RenderAsPng(FrameworkElement elt)
+		{
+			var renderSize = elt.RenderSize;
```

Comment:
Did you try using RenderTargetBitmap instead for this method? Or is it the encoder update that is not enough to get it running ?

Reply:
RenderTargetBitmap works correctly. I just followed the directions of this line:

https://github.com/workgroupengineering/uno/blob/82ef727242bdfa4e901b186b0d63aae77698a703/src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Shapes/Basic_Shapes.xaml.cs#L307